docs/models/index.rst

```diff
@@ -72,7 +72,7 @@ more-complicated cases, such as :doc:`convolution models
 Combining models
 ================
 
-Models can be combined and shared by using the standard Python
+The easiest way to combined and share models is by using the standard Python
```

Comment:
```suggestion
The easiest way to combine and share models is by using the standard Python
```

[I am also not 100% happy with my "shared/share" text but I wanted some way to say you can "re-use" components and "share" was the word I came up with; but that's somewhat irrelevant here]